content/it/docs/home/_index.md

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
If possible, scope down the current PR to the https://kubernetes.io/docs/concepts/overview/components/ page and move everything else in a separated PR